katana/native/file_manager/file_manager_utils/backend.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python3
+import os
+import shutil
+from katana.utils import user_utils
+
+def getpath(request):
+    """
+    This function is to get the path of the User-data location.
+    Whatever is pointed to by this path is displayed.
+    Currently, a static path is read from the file in my VM and the contents are displayed.
+    The file is opened and the path is read
+
+    :return: The path to user-data space
+    """
+    Object = user_utils.UserData('File Manager')
+    print(Object.get_user_path(request))
+    return Object.get_user_path(request)
+
+def delete(path):
+    """
+    This will check if it's a file or directory and then delete them.
+
+    :param path: Full path to File or Directory which has to be deleted.
+    :return: A string saying success or error
+    """
+    if (os.path.isdir(path)):
+        try:
+            shutil.rmtree(path)
+            return 'success'
+        except OSError:
+            print('Cannot delete Directory')
+    elif (os.path.isfile(path)):
+        try:
+            os.remove(path)
+            return 'success'
+        except OSError:
+            print('Cannot delete File')
+
+
+
+def rename(file_path, old_name, new_name):
+    """
+    This function will rename the file or directory.
+
+    :param file_path: The full path of the file or directory
+    :param old_name: Old Name of the file or directory
+    :param new_name: New Name of file or directory
+    :return: String containing success or error.
+    """
+    new_file_path = os.path.join(os.path.dirname(file_path), new_name)
+    try:
+        os.rename(file_path, new_file_path)
+    except os.error as e:
+        print(str(e))
+        return str(e)
+    return 'success'
```

Comment:
Please return booleans instead. If you also want to add more information into the return, return a message along with the boolean status. Eg ``` return True, "Renaming was successful" ```

Reply:
Done.